src/Gdi32.Desktop/Gdi32.cs

```diff
@@ -23,10 +23,30 @@ public static partial class Gdi32
         [DllImport(nameof(Gdi32), SetLastError = false)]
         public static extern int GetDeviceCaps(
             User32.SafeDCHandle hdc,
-            int nIndex);
+            DeviceCap nIndex);
```

Comment:
I think the strong-typing is a great idea. But yes, please add it as another overload to avoid breaking compat.